soccer/CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ set(ROBOCUP_LIB_SRC
     "motion/TrapezoidalMotion.cpp"
     "NewRefereeModule.cpp"
     "planning/CompositePath.cpp"
-    "planning/DirectTargetPathPlanner.cpp"
+    #"planning/DirectTargetPathPlanner.cpp"
```

Comment:
Well we can't merge something into master if it breaks pivot, that's a pretty essential motion :/

Can't we refactor the commands in a way that allows them to still be interpreted as before (MotionControl handles pivot, not a special planner)?  It's ok if this PR doesn't bring the structure of our commands entirely to how we want to have them in the end.  We may have to modify the types, etc a little bit later once the pieces are in-place.